### PR TITLE
Fixes both tests in StudioLmsAdvancedComponentTest

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
jenkins run e2e

**Comment by raeeschachar** [comment]
Please review. https://screencast.com/t/kuOn3KeOO
Fixes these tests https://build.testeng.edx.org/job/edx-e2e-tests/167/testReport/

@cpennington FYI

**Comment by raeeschachar** [comment]
I saw duplication of Verifies that user can add HTML components on Studio and LMS in [edx-platform tests](https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/studio/test_studio_components.py#L104). As per discussion will remove this tests from e2e tests in another PR.

**Comment by jzoldak** [comment]
Discussed in chat. Let's remove this test as it belongs in the edx-platform repo (if at all) instead of here.
Closing in favor of #127.

File: regression/tests/common/test_html_component.py [hunk 1]
```diff
@@ -95,14 +97,15 @@ def test_html_components(self):
...
             'Announcement',
-            'Anonymous User ID',
-            'Full Screen Image Tool',
             'IFrame Tool',
-            'Zooming Image Tool',
             'Raw HTML'
         ]
-        # Add components
-        add_components(self.unit_container_page, 'html', components)
+        # Add components. A BrokenPromise probably means that we missed it.
```
Comment (jzoldak) [inline regression/tests/common/test_html_component.py]: nit: a BrokenPromis probably means that we missed the notification.

**Comment by jzoldak** [comment]
@raeeschachar apologies, I had closed this by accident. Good to merge.

**Pull request merged**